src/compiler/axis.js

```diff
@@ -101,9 +101,9 @@ axis.grid = function(def, name, encoding, layout) {
           value: def.offset
         },
         x2: {
+          field: {group: 'mark.group.width'},
```

Comment:
@arvind  Thanks for your comment.  

Note that this is under `axis.properties.grid` and I try to reference the `group`’s width here.  

From testing both `field: {group: ‘mark.group.width’}` and `field: {group: ‘width’, level: 2}` works correctly but I kinda feel that the former reads better given that there is no documentation for how to refer to group’s width in axis’s properties.   At least the former give better clue that it is using parent mark’s group’s width.  

A few comments:
- People will have no clue what going up 2 levels in `axis.properties.grid` refers to the parent's `mark`.  If you think this is a preferred method, it should be documented in `axis.properties`’s docs and I'd be happy to change the output to use `field: {group: ‘width’, level: 2}`.  
- The `level` property looks confusing with `group`.  When using `level` with `parent` like shown in the example  `"field": {"parent": "f", "level": 2}`, it makes sense.  But `field: {group: ‘width’, level: 2}` looks confusing what `level` means.  That said, it might be too verbose and too late to name it `parentLevel`.